src/api_manager/method_impl.cc

```diff
@@ -112,7 +112,10 @@ void MethodInfoImpl::process_backend_rule(
     const ::google::api::BackendRule &rule) {
   backend_address_ = rule.address();
   backend_path_translation_ = rule.path_translation();
-  backend_jwt_audience_ = rule.jwt_audience();
+  backend_jwt_audience_ = rule.address();
```

Comment:
Do not default to the `address`. If `jwt_audience` is empty, do not use a JWT to authenticate to the backend.

In the case where we want the `jwt_audience` to be equal to the `address`, it will be explicitly set to the same value as the address.

Reply:
Thanks. Updated.